client/src/components/dashboard/Dashboard.js

```diff
@@ -1,24 +1,37 @@
 import Container from "react-bootstrap/Container"
 
-import PageCard from "./PageCard"
-import Layout from '../layout/Layout'
 import './Dashboard.css'
 
 
-function PageContent() {
+function Dashboard() {

```

Comment:
prefer arrow functions to be consistent

Reply:
Do you mean using `const Dashboard = () => {` instead? I personally find using `function Deashboard() {` easier to understand since it's consistent with other languages and we don't really need it be a variable function in this particular case. Though I'm not an expert in JS and maybe doing it with arrow functions is better??

Reply:
Yes, if this is a component, React best practices recommend using the arrow function. Besides, it will support React Hook if you need them later. https://frontarm.com/james-k-nelson/when-to-use-arrow-functions/

Reply:
Damn I actually never knew this lol. Although on that second point, I believe you can use React Hooks within non-arrow functions as well. 
Also every major function in `thunder` branch have been built using `function()` and not `() => {}`. Is this something important that we should go back and refactor those things, or we just move forward with arrow functions from now on. I think the latter is more wise since we still have a lot of features to go.

Reply:
@Researchify/logiciel has changed all React files associated with our commits to arrow functions.

Reply:
Can you assign one of your teammates to do it @kdav108 as I think it should be a very simply refactor that shouldn't take too long? I personally think one thing we can do well is to maintain a clean code base at all times to avoid technical debt piling up and my bad for not picking up on not using the arrow functions before. I must have missed it when doing my review for that PR.